File: Webapi/Webapi/Models/Events/EventUpdate.dto.cs
namespace Webapi.Models.Events;

#pragma warning disable CS8618 // data transfer object
public class EventUpdateDto
{
  public string Title { get; set; }

  public string? Description { get; set; }

  public DateTime Start { get; set; }

  public DateTime? End { get; set; }

  public EventCategory? Category { get; set; }

  public string? Location { get; set; }
}

File: Frontend/src/app/shared/api/models/event-update-dto.ts
/* tslint:disable */
/* eslint-disable */
import { EventCategory } from './event-category';
export interface EventUpdateDto {
  category?: EventCategory;
  description?: null | string;
  end?: null | string;
  location?: null | string;
  start?: string;
  title?: null | string;
}
